### ISSUE TITLE
bug(prefer-meta): false positive on playwright test.describe

### ISSUE THREAD

**Issue by CHC383** [marcalexiei/eslint-plugin-zod-x#121]
### 🧩 Description

Maybe due to the recent find logic changes, the following code produces lint error:

> The `.describe()` method still exists for compatibility with Zod 3, but `.meta()` is now the recommended approach

However, the `describe` method here is not related to Zod.

### 🪜 How to reproduce

```ts
import { test } from "@playwright/test";

test.describe("test", () => {});
```

### ✅ Expected behavior

ESLint doesn't report errors.

### 💻 System info

```shell
System:
    OS: macOS 26.1
    CPU: (10) arm64 Apple M1 Pro
    Memory: 151.14 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 22.21.1 - /Users/xxx/.local/share/mise/installs/node/22.21.1/bin/node
    npm: 10.9.4 - /Users/xxx/.local/share/mise/installs/node/22.21.1/bin/npm
    pnpm: 10.23.0 - /Users/xxx/.local/share/mise/installs/node/22.21.1/bin/pnpm
    Deno: 2.5.6 - /opt/homebrew/bin/deno
  npmPackages:
    eslint: ^9.39.1 => 9.39.1 
    eslint-plugin-zod-x: ^1.11.0 => 1.11.
```

**Comment by marcalexiei** [comment]
🤕 I missed this rule during new find logic updates. 

I'll fix it soon.

**Comment by marcalexiei** [comment]
Fix available in [v1.11.1](https://github.com/marcalexiei/eslint-plugin-zod-x/releases/tag/v1.11.1).

Thanks for the report!
